### PR TITLE
Update grpc

### PR DESCRIPTION
**Pull request by msteffen**
Fix https://github.com/pachyderm/pachyderm/issues/3265

### CODE REVIEW

File: etc/proto/Dockerfile [hunk 1]
```diff
@@ -4,12 +4,16 @@ LABEL maintainer="msteffen@pachyderm.io"
...
+RUN wget https://github.com/google/protobuf/releases/download/v3.6.1/protoc-3.6.1-linux-x86_64.zip -O protoc.zip
 RUN unzip protoc.zip -d /
 RUN cp -r /include /bin
 
 RUN go get -u -v github.com/golang/protobuf/proto
 RUN go get -u -v github.com/gogo/protobuf/protoc-gen-gofast
+# RUN go get -d -u -v github.com/gogo/protobuf/protoc-gen-gofast \
+#  && cd ${GOPATH}/src/github.com/gogo/protobuf/protoc-gen-gofast \
+#  && git checkout origin/dev \
+#  && go install ./...
```
Comment (jdoliner) [inline etc/proto/Dockerfile]: Why are we adding these lines but commented out?

Comment (msteffen) [reply etc/proto/Dockerfile]: This is a mistake. I added these while debugging the `"golang.org/x/net/context"` vs `"context"` issue (https://github.com/pachyderm/pachyderm/issues/3265) and forgot to remove them. They're gone now

Comment (jdoliner) [reply etc/proto/Dockerfile]: Makes sense, can't approve to confirm because I already approved.

**Comment by msteffen** [comment]
This screen shows that the first travis build is in progress, but when I click through the link (to https://travis-ci.org/pachyderm/pachyderm/builds/466683207), Travis shows that the build is finished and succeeded. I suspect a message was dropped between Travis and GitHub, and as CI indicates that the build succeeded, I'm going to merge this PR

**Pull request merged**